tests/sqlparser_mysql.rs

```diff
@@ -1065,6 +1065,14 @@ fn parse_set_names() {
     assert_eq!(stmt, Statement::SetNamesDefault {});
 }
 
+#[test]
+fn parse_limit_my_sql_syntax() {
+    mysql().one_statement_parses_to(
```

Comment:
If your usecase is to recover the same syntax (as make sure `LIMIT 5, 10` is serialized as `LIMIT 5,10` you probably need to change this test to use `verified_statement` or `verified_query`: https://github.com/sqlparser-rs/sqlparser-rs/blob/main/src/test_utils.rs#L108

Reply:
It doesn't. Should I target that?

The original idea was just to have the correct semantic, not the exact same conversion. But I can work on having the same syntax, which would change the AST for limit and offset

Reply:
I don't think you need to expand the scope of the PR to include preserving the syntax on round trip, I just wanted to ensure that was clear

Reply:
Sure! But you have a point, I'll try to look into it to see if it's possible/meaningful to make it semantically the same as the original query.

Reply:
I agree with your assement that you would have to extend the struct somehow to distinguish between `LIMIT x OFFSET y` and `LIMIT y, x`